ui/CHANGELOG.md

```diff
@@ -8,6 +8,10 @@ All notable changes to the **Prowler UI** are documented in this file.
 
 - SAML login integration [(#8203)](https://github.com/prowler-cloud/prowler/pull/8203)
 
+### Security
+
+- Enhanced password validation to enforce 12+ character passwords with special characters, uppercase, lowercase, and numbers [(#XXXX)](https://github.com/prowler-cloud/prowler/pull/XXXX)
```

Comment:
PR number is missing = )

Reply:
<img src="https://media4.giphy.com/media/v1.Y2lkPWJkM2VhNTdlOHNxeWY3Zmw0aWg2NWp4OG9idDY0ZHRobzNod3loeWU3NXdkNGdhciZlcD12MV9naWZzX3NlYXJjaCZjdD1n/TJawtKM6OCKkvwCIqX/giphy.gif"/>